screens/Home.js

```diff
@@ -158,7 +59,10 @@ const HomeScreen = () => {
   const [refreshing, setRefreshing] = useState(false);
   const [tryAgain, setTryAgain] = useState(false);
 
-  // fetch data on mount
+  useEffect(() => {
+    AsyncStorage.removeItem('@posts').catch((err) => console.log(err));
```

Comment:
idk..maybe there's some rogue `@posts` wondering around somewhere 🤷‍♂️ 
so let's just make sure it's removed